config/vendor-api-v1.yml

```diff
@@ -591,8 +592,22 @@ components:
         uk_residency_status:
           type: string
           maxLength: 256
-          description: The candidate’s UK residency status indicating their right to work and study in the UK
+          description: The candidate’s UK residency status indicating their right to work and study in the UK. The candidate can provide details as free text for example "Settled status".
           example: UK Citizen
+          enum:
+          - UK Citizen
+          - Irish Citizen
+          - right_to_work_or_study_details
+          - Candidate needs to apply for permission to work and study in the UK
+          - Candidate does not know
+        fee_payer:
+          type: string
+          maxLength: 256
```

Comment:
I've had a look at the current set of UCAS fee payer codes, and I think it might be ok to limit it as this is the list:
```
UCAS Fee Payer
01 - Private finance
02 - SLC,SAAS,NIBd,EU,Chl,IoM
04 - Research Council
05 - DH/Regional Health
06 - UK Govt intl award
07 - Training Agency
08 - Other UK govt award
09 - International agency
10 - UK industry/commerce
90 - Other source
99 - Not Known
```